packages/manager/modules/veeam-enterprise/src/dashboard/controller.js

```diff
@@ -9,19 +9,13 @@ export default class VeeamEnterpriseDashboardCtrl {
     CucControllerHelper,
     CucFeatureAvailabilityService,
     VeeamEnterpriseService,
-    goToLicenseActivate,
```

Comment:
Those are not needed anymore ?

Reply:
They are in the binding of the component I created: packages/manager/modules/veeam-enterprise/src/dashboard/component.js
Creating the component was I think the easiest way to use the basic routing tracking.